includes/class-amp-story-post-type.php

```diff
@@ -682,6 +702,18 @@ public static function enqueue_block_editor_scripts() {
 			'ampStoriesFonts',
 			self::get_fonts()
 		);
+
+		wp_localize_script(
+			self::AMP_STORIES_SCRIPT_HANDLE,
+			'ampStoriesExport',
+			array(
+				'action'  => 'amp-story-export',
```

Comment:
I'm thinking maybe we can put this in a property since it gets repeated all over the place.

Reply:
+1 for a class constant (or else if more suitable)